codebase/web/modules/custom/danse_label_override/danse_label_override.module

```diff
@@ -0,0 +1,418 @@
+    /** @var string $entityType */
+    $entityType = $entity->getEntityTypeId();
+    /** @var string $entityBundle */
+    $entityBundle = $entity->bundle();
+    /** @var int $entityId */
+    $entityId = $entity->id();
+
+    /* **** ARGUMENTS SECTION **** */
+    // Arguments are variables that can be included in your buttons' labels
+
+    // Add a new translation argument which will be used later depending on
+    // the exception that needs to be covered.
+    
+    $args['@verb'] = $subscribeMode ? (string) t('Subscribe to') : (string) t('Unsubscribe from');
+    $args['@verb_time'] = $subscribeMode ? (string) t('Get notified when') : (string) t('Unsubscribe when');
+    $args['@verb_darwin'] = $subscribeMode ? (string) t('Subscribe') : (string) t('Unsubscribe');
+
+
+    $args['@subscriptionMode'] = $context['subscriptionMode'];
+
+    // have to determine node vs taxonomy term here because it's not clear when using @entityType without this statement
+    if ($entity->getEntityTypeId() == 'node' && $entityBundle == 'forum'){
+        $args['@entityType'] = 'topic'; 
+    } elseif ($entity->getEntityTypeId() == 'taxonomy_term' && $entityType == 'taxonomy_term'){
+        $args['@entityType'] = 'forum';
+    } else {
+        $args['@entityType'] = $entity->getEntityTypeId();
+    }
+    $args['@bundle'] = $entity->bundle(); // Entity Types contain BUNDLES that contain individual entities/nodes
+    $args['@entityId'] = $entity->id();     
+    /* **** END ARGUMENTS SECTION **** */
+
+    /*
+     * DLO Important Concept: 
+     * You have the various entity types: Comments, Content, and Taxonomies (and more).
+     * There are others and you can create your own custom ones, but these foundational ones will help you get started.
+     * So, you have an Entity Type                          (like content types, comment types, and taxonomies)
+     *                  Which contain Bundles               (Content Type > Article or Forum Topic or Blog Posts)
+     *                      Which has Individual Entities   (Forum Topic > How do I bake cookies?)
+     *                              Which have Comments     (How do I bake cookies? [topic] > Here's my family recipe [comment])
+     *                              and Related Content     (individual entities that, when created, share taxonomy terms with the subscribed entity)
+     *                                      (Related content is described in Comment #15 here: https://www.drupal.org/project/danse/issues/3175932#comment-14122869)
+     * 
+     * The next block of code is where most of the customization on your part needs to happen.
+     * This is a layered set of conditions that allow you fine-grained control to target the exact label you want
+     * on the entity you want.
+     * We begin with Comment Entities, then Content Entities, then
+     * the last else if 
+     */
+    if ($entity instanceof CommentInterface) {
```

Comment:
I think we don't need to handle Comment's type, so hence this if block can be avoided. 

+ we don't have `comment_forum` entitybundle, so it's needed